caps-api/caps.js

```diff
@@ -14,18 +15,20 @@ app.use(express.urlencoded({extended:true}));
 const PORT = process.env.PORT || 3001;
 
 let delivery = {
-    store: 'Jeremys Paperclips',
-    orderID: 7,
-    customer: 'Jimbo',
-    address: '123 Smith St'
+    clientID: '12345',
+    eventName: 'event',
+    messageID: '987',
+    store: store,
```

Comment:
need to define it, put quotes around it, or you can do `faker.company.companyName()`